tests/fixtures/exposure/baseline_test_exposures.yml

```diff
@@ -0,0 +1,398 @@
+version: 2
+exposures:
+- name: Customers,_Sum_of_Number_Of_Orders_and_Average_of_Number_Of_Orders,_Grouped_by_Most_Recent_Order_(day)
+  description: '### Visualization: Line
+
+
+    No description provided in Metabase
+
+
+    #### Metadata
+
+
+    Metabase Id: __1__
+
+
+    Created On: __2021-07-21T08:00:47.453351Z__'
+  type: analysis
+  url: http://localhost:3000/card/1
+  maturity: medium
+  owner:
+    name: Alexander Butler
+    email: alex@source.co
```

Comment:
Are you comfortable making your email address visible and searchable? Maybe replace with something ending in `@example.com` or something.

Reply:
oops good call, thanks for catching that